SimPy/tkconsole.py

```diff
@@ -16,8 +16,15 @@
 # TODO: blink or highlight matching brackets
 # TODO: delete the prompt when joining lines; allow a way to break lines
 
-from Tkinter import *
-import sys, string, traceback, types, __builtin__
+try:  # Python 3
+    from tkinter import *
+except:  # Python 2
```

Comment:
Is there a specific error we can catch here rather than just everything

Reply:
I'll check